R/djpr_pal.R

```diff
@@ -0,0 +1,164 @@
+  }
+
+
+  if (isFALSE(faded)) {
+    palette <- regular_palette(n)
+  }
+
+
+  palette
+}
+
+
+regular_palette <- function(n) {
+
+
+  if (n == 1) {
+    palette <- djprtheme::djpr_blue
+  } else if (n == "2a") {
+    palette <- c(djprtheme::djpr_pacific_blue,
+                 djprtheme::djpr_blue)
+  } else if (n == 2) {
+    palette <- c(djprtheme::djpr_blue,
+                 djprtheme::djpr_green)
+  } else if (n == 3) {
+    palette <- c(djprtheme::djpr_pacific_blue,
+                 djprtheme::djpr_cool_grey_1,
+                 djprtheme::djpr_cobalt)
+  } else if (n == 4) {
+    palette <- c(djprtheme::djpr_pacific_blue,
+                 djprtheme::djpr_blue,
+                 djprtheme::djpr_green,
+                 djprtheme::djpr_spray)
+  } else if (n == 5) {
+    palette <- c(djprtheme::djpr_pacific_blue,
+                 djprtheme::djpr_blue,
+                 djprtheme::djpr_green,
+                 djprtheme::djpr_spray,
+                 djprtheme::djpr_cool_grey_1)
+  } else if (n == 6) {
+    palette <- c(djprtheme::djpr_pacific_blue,
+                 djprtheme::djpr_cobalt,
+                 djprtheme::djpr_blue,
+                 djprtheme::djpr_green,
+                 djprtheme::djpr_spray,
+                 djprtheme::djpr_cool_grey_1)
+  } else if (n == 7) {
+    palette <- c(djprtheme::djpr_pacific_blue,
+                 djprtheme::djpr_cobalt,
+                 djprtheme::djpr_blue,
+                 djprtheme::djpr_green,
+                 djprtheme::djpr_spray,
+                 djprtheme::djpr_cool_grey_1,
+                 djprtheme::djpr_cool_grey_11)
+  } else if (n == 8) {
+    palette <- c(djprtheme::djpr_pacific_blue,
+                 djprtheme::djpr_cobalt,
+                 djprtheme::djpr_iris_blue,
+                 djprtheme::djpr_blue,
+                 djprtheme::djpr_green,
+                 djprtheme::djpr_spray,
+                 djprtheme::djpr_cool_grey_1,
+                 djprtheme::djpr_cool_grey_11)
+  } else if (n == 9) {
+    palette <- c(djprtheme::djpr_pacific_blue,
+                 djprtheme::djpr_cobalt,
+                 djprtheme::djpr_iris_blue,
+                 djprtheme::djpr_blue,
+                 djprtheme::djpr_green,
+                 djprtheme::djpr_spray,
+                 djprtheme::djpr_cool_grey_1,
+                 djprtheme::djpr_golden_yellow,
+                 djprtheme::djpr_cool_grey_11)
+  } else if (n == 10) {
+    palette <- c(djprtheme::djpr_pacific_blue,
+                 djprtheme::djpr_cobalt,
+                 djprtheme::djpr_iris_blue,
+                 djprtheme::djpr_blue,
+                 djprtheme::djpr_green,
+                 djprtheme::djpr_persian_green,
+                 djprtheme::djpr_spray,
+                 djprtheme::djpr_cool_grey_1,
+                 djprtheme::djpr_golden_yellow,
+                 djprtheme::djpr_cool_grey_11)
+  }
+  palette
+}
+
+djpr_colour_manual <- function(x){
+  scale_colour_manual(values=djpr_pal(n=x))
+}
+
```

Comment:
same as above - need to connect this function to the documentation (with the `#' @rdname` bit) and tell R to make the function available to users (`#' @export`)